draft-ietf-jsonpath-base.md

```diff
@@ -1500,22 +1500,18 @@ The abstract instances above can be obtained from the concrete representations i
 | `Nodes(nl)`       | A list of zero or more nodes, e.g., from a `filter-path` resulting in the nodelist `nl`, which may or may not be empty  |
 {: #tbl-typerep title="Concrete representations of abstract instances"}
 
-### Type Conversions {#type-conv}
+### Type Conversion {#type-conv}
 
-The following type conversions may occur:
+The following implicit type conversion may occur:
 
-* Where an expression with a declared type of `NodesType` needs to be
-  converted to a `ValueType`, the conversion proceeds as follows:
-  * If the nodelist contains a single node, the conversion result is
-    the value of the node.
-  * If the nodelist is empty or contains multiple nodes, the
-    conversion result is `Nothing`.
 * Where a member of `NodesType` needs to be converted to a
   `LogicalType`, the conversion proceeds as follows:
   * If the nodelist contains one or more nodes, the conversion result
     is `LogicalTrue`.
   * If the nodelist is empty, the conversion result is `LogicalFalse`.
 
+An expression with a declared type of `NodesType` can be converted to a `ValueType` using the explicit type conversion function `value()` (see {{value}}).
```

Comment:
I'm not even sure this is a "conversion" function (there may be several semantics for such a conversion function, and the merit of the present PR is that it makes that function explicit so others can be added later).

Reply:
Yes, let's avoid that terminology. Adopted your suggestion.